spec/e2e/end_to_end.rb

```diff
@@ -103,20 +103,18 @@ def fetch_addresses_balances_test(wallet)
 
 def transfer_test(imported_address, new_address)
   # Transfer gwei from imported address to new address.
-  puts 'Transfering 1 Gwei from imported address to new address...'
+  puts 'Transferring 1 Gwei from imported address to new address...'
   t = imported_address.transfer(1, :gwei, new_address).wait!
   expect(t.status).to eq('complete')
   puts "Transferred 1 Gwei from #{imported_address} to #{new_address}"
 
-  # Transfer some eth for gas fee to new address.
-  t2 = imported_address.transfer(0.00001, :eth, new_address).wait!
-  expect(t2.status).to eq('complete')
-  puts "Transferred 0.00001 Eth from #{imported_address} to #{new_address}"
+  # Fund the new address with faucet.
+  new_address.faucet
```

Comment:
Should we add a `puts` statement around the `faucet` here to track output?

Could be nice to expect that it does not raise an error (that way we at least have good log output?)